tools/visual-tester/src/runner.js

```diff
@@ -217,6 +258,16 @@ module.exports = class Runner {
         const query = options.selector || this.globalOptions.selector;
         await page.waitForSelector(query);
         const selector = await page.$(query);
+
+        if (options.removeFromDom && options.removeFromDom !== '') {
```

Comment:
I think that we could extract this as helper function and not be part of the internal API as options. 

And just let the `page` object be part of the test.